docs/source/examples/training/index.rst

```diff
@@ -14,6 +14,7 @@ Training
    Finetuning Llama 2 <llama-2-finetuning.md>
    NeMo <nemo.md>
    Ray <ray.md>
+   TorchTitan <torchtitan/README.md>
```

Comment:
Shall we also add to the table at https://github.com/skypilot-org/skypilot?tab=readme-ov-file#runnable-examples?